internal/graphql/resolver/operator.go

```diff
@@ -40,7 +41,20 @@ func (r *Resolver) changeOperatorStatus(ctx context.Context, status model.Status
 	}
 
 	go func(del bool, kubeclient *mesherykube.Client) {
-		err := initialize(kubeclient, del)
+		err := r.cleanEntries(del)
+		if err != nil {
+			r.Log.Error(err)
+			r.operatorChannel <- &model.OperatorStatus{
+				Status: status,
+				Error: &model.Error{
+					Code:        "",
+					Description: err.Error(),
```

Comment:
@kumarabd should this be filled-in?

Reply:
We don't have the code at the moment. So no...